### ISSUE TITLE
Allow to atlas metallics and roughness maps

### ISSUE THREAD

**Issue by TheCreeperCow** [Grim-es/material-combiner-addon#23]
Check if a material has a metallics or roughness set to them and if its a image if it is atlas those as well as i have to manually make them for now which is lets just say less than ideal

**Comment by AraHaan** [comment]
I do not think there is a need to atlas metallics and roughness maps because they are a shader thing no? A think not directly loaded to vram.

**Comment by RedSpeeds** [comment]
The workflow intended here is that in substance you make the metallics and roughness maps and then you load them into blender then in CATS you combine them thru your addon and id be nice if the metallics and roughness set on those materials can be atlassed as well so that we can move the atlas into unity for the final shader

**Comment by AraHaan** [comment]
> The workflow intended here is that in substance you make the metallics and roughness maps and then you load them into blender then in CATS you combine them thru your addon and id be nice if the metallics and roughness set on those materials can be atlassed as well so that we can move the atlas into unity for the final shader

Easier said then done with a mesh uses:
- RGB node that gets atlas'd as 10x10 px
- has a metallic and roughness map of greater than that size in pixels as images.

The extension messes with the UV on it somewhat, how will it know to grab it at the proper place in the metallic, roughness, and alpha maps that can be at any big size btw?

**Comment by RedSpeeds** [comment]
The Metallic and roughness map would be a separate image. Im not too much aware of the technical details of this personally

**Comment by AraHaan** [comment]
> The Metallic and roughness map would be a separate image. Im not too much aware of the technical details of this personally

Yes, but it would have to be aware of the UV as well and the UV can be problematic for some people.

**Comment by OVERMEGAKILL** [comment]
Hello so the fix for the pbr toon1 metallic roughness missing is to do a map, any idea on how to proceed that i'm a newbie in this i'm learning the hard way any help is welcomed
